src/index.js

```diff
@@ -4,7 +4,7 @@ const { createRouter } = require('./app');
 module.exports.Satellite = require('./satellite');
 module.exports.logger = require('./logger');
 module.exports.hash = require('./hash');
-module.exports.createError = require('http-errors');
+module.exports.createError = require('./createError').createError;
```

Comment:
`module.exports.createError = require('./create-error');`

NOTE: always name files in lower-case, since not all filesystems are case sensitive.